clients/scanoss/src/main/kotlin/model/Model.kt

```diff
@@ -67,6 +67,21 @@ data class License(
     val osadlUpdated: String? = null
 )
 
+/**
+ * The "copyrights" section of the raw report.
+ */
+@Serializable
+data class Copyright(
```

Comment:
This commit does not than just documeting an (existing) `Copyright` class, it introduces the `Copyright` class:

    scanoss: Introduce a dedicated `Copyright` class